hd-render-manager.c:3414:46: error: 'HdRenderManagerPrivate' {aka 'struct _HdRenderManagerPrivate'} has no member named 'zoomed'; did you mean 'home'?
     g_return_if_fail (!render_manager->priv->zoomed);
                                              ^~~~~~
--
diff --git a/src/home/hd-render-manager.c b/src/home/hd-render-manager.c
index b3a2943..8c1ceca 100644
--- a/src/home/hd-render-manager.c
+++ b/src/home/hd-render-manager.c
@@ -3411,7 +3411,6 @@ hd_render_manager_press_effect (void)
     HdRenderManagerPrivate *priv = render_manager->priv;
 
     g_return_if_fail (render_manager != NULL);
-    g_return_if_fail (!render_manager->priv->zoomed);
 
     if (!priv->press_effect)  
     {
